compiler/damlc/daml-doc/src/DA/Daml/Doc/Extract.hs

```diff
@@ -123,9 +123,14 @@ extractDocs extractOpts diagsLogger ideOpts fp = do
 
             (adts, md_templateInstances) =
                 partitionEithers . flip map filteredTyCons $ \adt ->
-                    case getTemplateInstanceDoc adt of
+                    case find (\td -> td_name td == ad_name adt) md_templates of
```

Comment:
@associahedron I’ve hacked around this for now since I don’t want to clutter this PR too much but we should kill the generic templates support from `daml-docs` in a separate PR.

Reply:
Makes sense. I can do it if you don't mind.